src/http/mod.rs

```diff
@@ -20,3 +20,8 @@ pub use request_template::RequestTemplate;
 pub use response::*;
 
 pub use crate::app_context::AppContext;
+
+#[derive(Default, Clone, Debug)]
+pub struct HttpFilter {
+    pub on_request: String,
+}
```

Comment:
Can add a bit of documentation about what it does.

Reply:
sure.